tests/unit_tests/gui/conftest.py

```diff
@@ -436,7 +436,17 @@ def wait_for_child(gui, qtbot: QtBot, typ: Type[V], *args, **kwargs) -> V:
     return get_child(gui, typ, *args, **kwargs)
 
 
-def get_child(gui, typ: Type[V], *args, **kwargs) -> V:
+def get_child(gui: QWidget, typ: Type[V], *args, **kwargs) -> V:
     child = gui.findChild(typ, *args, **kwargs)
     assert isinstance(child, typ)
     return child
+
+
+from typing import List
```

Comment:
Propose to move imports to the top?

Reply:
Yes, I have no idea why the style linter did not catch this.